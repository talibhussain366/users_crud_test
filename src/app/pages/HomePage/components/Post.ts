import styled from 'styled-components/macro';

export const Post = styled.div`
  color: white;
  padding: 10px;
  margin-top: 10px;
  border: 1px solid white;
  border-radius: 5px;
`;
