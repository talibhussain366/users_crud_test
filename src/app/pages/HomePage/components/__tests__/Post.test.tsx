import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { Post } from '../Post';

const shallowRenderer = createRenderer();

describe('<Post />', () => {
    it('<Post />', () => {
        shallowRenderer.render(<Post />);
        const renderedOutput = shallowRenderer.getRenderOutput();
        expect(renderedOutput).toMatchSnapshot();
    });
});
