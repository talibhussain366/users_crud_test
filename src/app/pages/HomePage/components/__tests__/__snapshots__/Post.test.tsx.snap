// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> <Post /> 1`] = `
<div
  className="Post-sc-1lki2ve-0 eiYnZO"
/>
`;
